k8s/tf/200_single_state/eu-central-1/eks.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   cluster_features = {
-    "aws_calico"         = true
+    # "aws_calico"         = true # added using helm, ref: https://docs.aws.amazon.com/eks/latest/userguide/calico.html
```

Comment:
Why is calico commented out?

Reply:
I removed calico installed from the EKS module and installed a most recent version using helm following the doc wondering it could help on requests hanging, it didn't change at the end, that's why I just updated in eu-central-1, and keep in us-west-2 as is.